<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolicyConfig</class>
 <widget class="QDialog" name="PolicyConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="dayOfTheWeekLayout">
     <property name="title">
      <string>Day Of The Week</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="dayOfTheWeekButton_0">
        <property name="text">
         <string>Sun</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dayOfTheWeekGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dayOfTheWeekButton_1">
        <property name="text">
         <string>Mon</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dayOfTheWeekGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dayOfTheWeekButton_2">
        <property name="text">
         <string>Tue</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dayOfTheWeekGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dayOfTheWeekButton_3">
        <property name="text">
         <string>Wed</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dayOfTheWeekGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dayOfTheWeekButton_4">
        <property name="text">
         <string>Thu</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dayOfTheWeekGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dayOfTheWeekButton_5">
        <property name="text">
         <string>Fri</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dayOfTheWeekGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dayOfTheWeekButton_6">
        <property name="text">
         <string>Sat</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dayOfTheWeekGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="timeLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="startTimeLayout">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="start_time_label">
         <property name="text">
          <string>Start Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PolicyTimeEdit" name="start_time">
         <property name="displayFormat">
          <string>hh:mm</string>
         </property>
         <property name="time">
          <time>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="endTimeLayout">
       <item>
        <widget class="QLabel" name="end_time_label">
         <property name="text">
          <string>End Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PolicyTimeEdit" name="end_time">
         <property name="displayFormat">
          <string>hh:mm</string>
         </property>
         <property name="time">
          <time>
           <hour>12</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hostListLayout">
     <item>
      <widget class="QListWidget" name="hostList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PolicyTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>policy/policytimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="dayOfTheWeekGroup"/>
 </buttongroups>
</ui>
